packages/node_modules/@webex/plugin-meetings/src/common/collection.js

```diff
@@ -0,0 +1,75 @@
+import {MEETINGS} from '../constants';
+
+/**
+ * @class Collection
+ */
+export default class Collection {
+  namespace = MEETINGS;
+
+  /**
+   * @param {String} property
+   * @memberof Collection
+   */
+  constructor(property) {
+    this.propertyName = property;
+    this[this.propertyName] = {};
+  }
+
+  /**
+   * @param {String} id ID of the thing stuffed into the collection at id location
+   * @returns {Any} returns whatever is being stuffed into the collection
+   * @public
+   * @memberof Collection
+   */
+  get(id) {
+    if (this[this.propertyName] && this[this.propertyName][id]) {
+      return this[this.propertyName][id];
+    }
+    return null;
+  }
+
+  /**
+   * @param {String} id the id of the meeting info instance to add to the collection
+   * @param {Any} set the thing to set in the collection
```

Comment:
Probably name this something other than the function name, e.g., `somethingValue`.

Reply:
okay will change